src/cares_wrap.cc

```diff
@@ -953,34 +363,34 @@ int ParseTxtReply(Environment* env,
       if (!txt_chunk.IsEmpty()) {
         if (need_type) {
           Local<Object> elem = Object::New(env->isolate());
-          elem->Set(context, env->entries_string(), txt_chunk).Check();
-          elem->Set(context,
+          elem->Set(env->context(), env->entries_string(), txt_chunk).Check();
+          elem->Set(env->context(),
                     env->type_string(),
                     env->dns_txt_string()).Check();
-          ret->Set(context, offset + i++, elem).Check();
+          ret->Set(env->context(), offset + i++, elem).Check();
         } else {
-          ret->Set(context, offset + i++, txt_chunk).Check();
+          ret->Set(env->context(), offset + i++, txt_chunk).Check();
         }
       }
 
       txt_chunk = Array::New(env->isolate());
       j = 0;
     }
 
-    txt_chunk->Set(context, j++, txt).Check();
+    txt_chunk->Set(env->context(), j++, txt).Check();
   }
 
   // Push last chunk if it isn't empty
   if (!txt_chunk.IsEmpty()) {
     if (need_type) {
       Local<Object> elem = Object::New(env->isolate());
-      elem->Set(context, env->entries_string(), txt_chunk).Check();
-      elem->Set(context,
+      elem->Set(env->context(), env->entries_string(), txt_chunk).Check();
+      elem->Set(env->context(),
                 env->type_string(),
                 env->dns_txt_string()).Check();
-      ret->Set(context, offset + i, elem).Check();
+      ret->Set(env->context(), offset + i, elem).Check();
     } else {
-      ret->Set(context, offset + i, txt_chunk).Check();
+      ret->Set(env->context(), offset + i, txt_chunk).Check();
```

Comment:
Just going to point out that these changes will all have to be undone if we add support for multi-context builtins

Reply:
Is there any work happening on that right now? And would that also be true for many large chunks of core code? And would that even be relevant here if the `getdns` alternative impl lands? (I have to assume that multi-context builtins will take quite a bit longer to land than the `getdns` impl and that we likely wouldn't backport it to 14 or 16).

Reply:
> Is there any work happening on that right now? 

Not me, at least. 

> And would that also be true for many large chunks of core code? 

Probably, yes.

> And would that even be relevant here if the `getdns` alternative impl lands? (I have to assume that multi-context builtins will take quite a bit longer to land than the `getdns` impl and that we likely wouldn't backport it to 14 or 16).

I mean, it would affect that implementation as well, yes.

Reply:
> I mean, it would affect that implementation as well, yes.

Will worry about that when it becomes an issue then :-)